bgpd/bgp_io.c

```diff
@@ -336,7 +336,24 @@ static uint16_t bgp_write(struct peer *peer)
 				BGP_EVENT_ADD(peer, TCP_fatal_error);
 				SET_FLAG(status, BGP_IO_FATAL_ERR);
 			} else {
-				SET_FLAG(status, BGP_IO_TRANS_ERR);
+				_Atomic uint32_t last_write;
```

Comment:
You don't need `_Atomic` on the local variables you read into, just on the long-lived datastructures that are accessed concurrently

(Doesn't really break anything, might make this code slightly slower.)

Reply:
fixed.